README.md

```diff
@@ -137,7 +150,7 @@ sudo apt-get install \
 
 The above packages are sufficient for Ubuntu 13.10 and Ubuntu 14.04.
 
-In the folly directory, run
+In the folly/folly directory, run
```

Comment:
This bit is not actually correct - the repo may be cloned to any place, not only to a directory named `folly`.